### DISCUSSION TITLE
[v1]How can I change the position, size, or floating style of the title bar on Mac?

### DISCUSSION THREAD

**Discussion by nongyehong** [tauri-apps/tauri#10503] [Q&A] [answered]
![97bce0291bd947bff5d88b4f22b8f98a](https://github.com/user-attachments/assets/de33559d-f154-4773-a41c-40e401b1ee7f)
I want to make such an effect, what should I do, help, I'm a front-end developer, and rust is a bit rusty

**Comment by 1424784560** [comment]
We have the same problem

**Reply by nongyehong** [reply, marked as answer]
I solved the problem,Just set the titleBarStyle: 'Overlay' and hiddenTitle: true properties, which are unique to Mac in tauri.conf_json
